packages/shared/src/components/checklist/ChecklistStep.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { IconSize } from '../Icon';
+import ArrowIcon from '../icons/Arrow';
+import { ChecklistStepProps } from '../../lib/checklist';
+import ChecklistAIcon from '../icons/ChecklistA';
+import classed from '../../lib/classed';
+
+const Container = classed('div', '');
+
+const ChecklistStep = ({
+  className = {},
+  step,
+  isOpen = false,
+  isActive = false,
+  onToggle,
+  children,
+}: ChecklistStepProps): ReactElement => {
+  const isCompleted = !!step.action.completedAt;
+
+  return (
+    <Container className={className.root}>
```

Comment:
Ah sorry think you misinterpreted my previous comment, I mean instead of `root` we use `container` normally. Not specifically the div :)